app/models/league.rb

```diff
@@ -37,10 +37,6 @@ def update_matches # rubocop:disable Metrics/AbcSize
         away_goals: m.away_goals,
         result: m.result
       )
-      if match.changed?
-        Rails.logger.info "Updating #{match.team_home.name} x #{match.team_away.name}..."
-        match.save
-      end
```

Comment:
If you want, you can preserve the logging on the successful outcome with update by checking if the returned value of `update` is truthy.


```rb
updated = match.update(
  status: m.status,
  date: m.date,
  home_goals: m.home_goals,
  away_goals: m.away_goals,
  result: m.result
)

if updated
  Rails.logger.info "Updating #{match.team_home.name} x #{match.team_away.name}..."
else
  Rails.logger.info "Failed to update Match #{match.id}:\n\t#{e.errors.full_messages.join("\n"}"
end
```

Reply:
Resolved in #17